test/utils/hooks.ts

```diff
@@ -28,6 +47,11 @@ declare module 'mocha' {
   }
 }
 
+interface GlobalFixturesContext {
```

Comment:
This should be removed. Seems like. Only and interface with `compose` below is needed

Reply:
Done